add.go

```diff
@@ -43,10 +43,16 @@ func addUrl(destination, srcurl string) error {
 // Add copies contents into the container's root filesystem, optionally
```

Comment:
I might be light on the amount of tea that I've had today, but I'm struggling with this comment.  Should it be "Add the source's contents into the containter's root ..."